Sources/Controllers/AnimatableTabBarController.swift

```diff
@@ -65,3 +78,28 @@ open class AnimatableTabBarController: UITabBarController, TransitionAnimatable
     delegate = navigator
   }
 }
+
+fileprivate extension UITabBar {
+
+    func view(for item: UITabBarItem) -> UIControl? {
+      // return item.value(forKeyPath: "view") as? UIControl // apple could not allow that
+      guard let items = self.items else {
+        return nil
+      }
+      guard let index = items.index(of: item) else {
+        return nil
+      }
+      // get all buttons
+      // 1/ filter on control, not safe if apple add a new control in bar
+      let controls = self.subviews.filter { $0 is UIControl }
+      // 2/ filter on name, not safe if button class change
+      // let controls = self.subviews.filter { String(describing: type(of: $0 )) == "UITabBarButton" }
```

Comment:
Both ways are hacky. I have seen a lot of code is using `UITabBarButton`, it is more specific than `UIControl`. But either way, I am fine.